### ISSUE TITLE
"cxx14" is not a recognized standard, fails heuermh/adam build

### ISSUE THREAD

**Issue by heuermh** [heuermh/docker-linuxbrew-bio#6]
Latest heuermh/adam DockerHub build failed due to problems in the homebrew-science tap
```
==> Tapping homebrew/science
Cloning into '/home/linuxbrew/.linuxbrew/Library/Taps/homebrew/homebrew-science'... 
Error: Invalid formula:
/home/linuxbrew/.linuxbrew/Library/Taps/homebrew/homebrew-science/kmc.rb 
"cxx14" is not a recognized standard

Error: Cannot tap homebrew/science: invalid syntax in tap!
Removing intermediate container c88a4a5ea685

The command '/bin/sh -c brew tap homebrew/science && brew install adam' returned
a non-zero code: 1
```

**Comment by alanhoyle** [comment]
Per conversation referenced elsewhere, I think fixing this is dependent on upstream Linuxbrew Docker (either linuxbrew/linuxbrew or sjackman/linuxbrew) progressing to version 1.1.11+.  right, @heuermh and @sjackman?  

Note that this issue also causes failures on many other docker-linuxbrew-bio builds including snpeff, freebayes, etc.

**Comment by sjackman** [comment]
I've tagged `brew` 1.1.11 just now. `brew update` should resolve this issue.

**Comment by heuermh** [comment]
Fixed by commit 2d7f0e5ee1c27aba81e6a900a8de3ae877860821.  Thank you, @sjackman @alanhoyle
